src/vm/analysis/arithmetic_checker/tests.rs

```diff
@@ -130,95 +130,123 @@ fn test_variables_fail_arithmetic_check() {
 }
 
 #[test]
-fn test_functions() {
-    let bad_tests = [
+fn test_functions_clarity1() {
```

Comment:
note: I separated these tests into "clarity1" and "clarity2", similar to what was suggested in https://github.com/blockstack/stacks-blockchain/pull/2735

Reply:
Thanks for doing this -- I think it's a healthy habit to maintain, especially since there will likely be future versions of Clarity.